docs/userGuide/syntax/lists.md

```diff
@@ -27,11 +27,11 @@
 * Item 2
   * Item 2.1 { icon="fas-file-code" }
   * Item 2.2
-* Item 3 { icon="fas-code-branch" }
+* Item 3 { text="3" }
```

Comment:
Also refer to https://github.com/MarkBind/markbind/pull/2316

I think you are not showcasing what is this supposed to be used for `{ text="Step 2", class="text-warning" }`. (Sort of like using step2 as an icon)

From my understandings, it seem that this is to use text in place of icons (so I think you are not showcasing what this feature is for).

Reply:
<img width="290" alt="Screenshot 2024-03-10 at 4 17 02 PM" src="https://github.com/MarkBind/markbind/assets/59471320/ab46f191-ad92-43fc-bfa2-ed4af9ccfb9a">

Agree that this isn't really super clear that text is being used in place of an icon. It just kinda looks like the icon is removed?

Reply:
This feature is to add the text next to the icon though - it doesn't replace the icon. 
I was thinking that it would be better if users can use both concurrently which is why there is the icon and then the text on the right. 
What do you all think? Is it better to replace the icon instead?

Reply:
@yucheng11122017 
I thought it is to replace the icon.
![image](https://github.com/MarkBind/markbind/assets/75515229/e49195fa-ed08-4c63-97c0-e009085d50ea)

Reply:
Ok fair enough

I would prefer to leave the functionality like this though to allow users to use both concurrently